charts/osm/templates/osm-gateway-service.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.OpenServiceMesh.featureFlags.enableMulticlusterMode }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: osm-gateway
+  namespace: {{ include "osm.namespace" . }}
+  labels:
+    {{- include "osm.labels" . | nindent 4 }}
+    app: osm-gateway
+spec:
+  ports:
+    - name: admin-port
+      port: 15000
+      targetPort: 15000
```

Comment:
admin port should never be exposed as a service port. It's only meant for admin interaction while debugging.